itou/approvals/models.py

```diff
@@ -788,6 +788,10 @@ class Meta:
                     ("approval", RangeOperators.EQUAL),
                 ),
             ),
+            models.CheckConstraint(
+                check=Q(start_at__lte=TruncDate(Now())),
+                name="cannot_start_in_the_future",
```

Comment:
```suggestion
                name="$(class)s_cannot_start_in_the_future",
```
Pour n'avoir que des avantages :
- Facilement identifier la contrainte en cas d'erreur.
- Éviter les collisions de nom.
- Bonus : Permet l'héritage sans modification future.

Reply:
Je ne savais pas que cette syntaxe existait. Merci :+1: